tech/post/2017-11-06-Kubernetes-Taints&Tolerations.md

```diff
@@ -0,0 +1,302 @@
+# Taints and Tolerations
+
+## 1. Introducation
+
+Taints and tolerations work together to ensure that pods are not scheduled onto inappropriate nodes.
+One or more taints are applied to a node; this marks that the node should not accept any pods that do not tolerate the taints.
+Tolerations are applied to pods, and allow the pods to schedule onto nodes with matching taints.

```

Comment:
S/allow the pods to schedule onto/allow the pods to be scheduled onto

Reply:
Done